### ISSUE TITLE
Imported users cannot log in

### ISSUE THREAD

**Issue by kabelo38** [YetiForceCompany/YetiForceCRM#4028]
#### Issue

I have exported users from v3.4.237 and imported the users to  v4.0.0 clean install but however none of the imported users can log in.

#### Actual Behavior

I get incorrect username or password

#### Expected Behavior

User to log into the dashboard

#### How to trigger the error

1. Go CRM URL
2. Put in your credentials

#### Your Environment

* YetiForce Version used: v4.0.0
* Browser name and version:
* Environment name and version:Linux
* Operating System and version:Ubuntu 16

**Comment by migoi** [comment]
something in the log file? (\<your-yeti-path\>/cache/logs/phpError.log)

**Comment by kabelo38** [comment]
Nope none

I allowed cron - PRIVILEGES_UPDATER to run over night hoping it is related to it but 24 hours later still no way forward.

**Comment by kabelo38** [comment]
okay cool i got it working the problem is if you import users the passwords and existing hash do not get imported they are regenerated so to get it working i had to go to new db under vtiger_users and replaced user_password,user_hash and  confirm_password fields with old values to get it right alternative if u have many users just right a script to update vtiger_users table.

**Comment by ldgbc** [comment]
@kabelo38 Hi Kabelo38, thank you for your tip. That could be the case however this bug error on a fresh installation with only "admin" as the user.

How I tested this bug is as follow:
1) extract yetiforce 3.4 to a test directory
2) rename it to match my subdomain.website.com
3) open subdomain.website.com
4) install
5) Setting > Logs > Update > Upload 3.4.1472 > Update complete.
6) Logout
7) Try to log back in but it always fail. It might be possible that hash was changed during update so I'll try that after this post.

So I'm still leaning toward this, "MYSQLND" libraries as my Shared Hosting does not have that installed.
